src/test/game/Language/ItalianTest.java

```diff
@@ -0,0 +1,59 @@
+package test.game.Language;
+
+import main.game.Language.Italian;
+import org.junit.Test;
+
+import static main.game.Moves.Moves.*;
+import static org.junit.Assert.assertEquals;
+
+public class ItalianTest {
+
+    @Test
+    public void promptPlayerForMove() {
+        Italian italian = new Italian();
```

Comment:
A minor thing: you're creating an instance of `Italian` for each test, you could do this only once with @BeforeAll and create an instance variable (you can find an example [here](https://github.com/Gabbendorf/Java-Roman-Numerals-Kata/blob/master/src/test/RomanNumeralsTest.java))